### PR TITLE
RD-2455 Transform markdown links inside the styled message blocks into the anchor elements

### PR DESCRIPTION
**Pull request by NorbertAmartus**
## Description

Currently the links in the documentation (which are displayed in form of `info` | `warning` | `tip` message) are not being displayed properly.
Changes provided by this PR solves this issue, by transforming markdown links into the anchor elements

## Screenshots / Videos

Before
before

After
After

## Checklist

- [x] My code follows the UI Code Style.
- [ ] I verified that all tests and checks have passed.
- [x] I verified if that change requires a documentation update.
- [x] I added proper labels to this PR.

## Tests

 

I've executed the test file, which was mentioned inside the [jira ticket](https://cloudifysource.atlassian.net/browse/RD-2455) (that it connected to this PR) and it has passed successfully 🎉 

## Documentation

N/A

### CODE REVIEW

**Review by qooban — COMMENTED** [review]
Widget's help description looks better now, but you'll probably need a change in how README.md files are interpreted, as right now link is broken:

https://user-images.githubusercontent.com/5202105/146376747-78f615ec-b65c-4b38-987a-d050f3214e73.mp4

Hint: Check `updateReadmeLinks` in `app/utils/widgetDefinitionsLoader.ts`.

File: scripts/updateReadmes.js [hunk 1]
```diff
@@ -9,10 +9,14 @@ const i18nBundlePath = '../app/translations/en.json';
 const supportedParams = require(i18nBundlePath).widgets.common.readmes.params;
 
 function log(prefix, message) {
+    return;
+    // eslint-disable-next-line
```
[the remark this answers is not in the document]
Comment (NorbertAmartus) [reply scripts/updateReadmes.js]: Yeah 😁 
I've created a proper commit ... but I didn't push it 😢 

I've pushed it now, and the issue has "dissapear" 😁

**Comment by NorbertAmartus** [comment]
> Widget's help description looks better now, but you'll probably need a change in how README.md files are interpreted, as right now link is broken:
> 
>  Peek.2021-12-16.14-00.mp4 
> Hint: Check `updateReadmeLinks` in `app/utils/widgetDefinitionsLoader.ts`.

Thanks for mentioning that issue, and also for having the eye of an eagle! 🦅 
Problem has been solved! 🎉

**Review by qooban — APPROVED** [review]
Works good 👍🏻

**Pull request merged**